packages/grafana-ui/src/components/ThresholdsEditorNew/ThresholdsEditor.tsx

```diff
@@ -59,7 +59,11 @@ export class ThresholdsEditor extends PureComponent<Props, State> {
       nextValue = steps[steps.length - 1].value + 10;
     }
 
-    const color = colors.filter((c) => !steps.some((t) => t.color === c))[1];
+    let color = colors.filter((c) => !steps.some((t) => t.color === c))[1];
+    if (!color) {
+      // Default color when all colors are used
+      color = '#CCCCCC';
```

Comment:
Maybe we should use the primary text color from the theme but that would mean you need to extend the component with theme and that might not be worth the while for this small change.